Creazione classe Pedina #12

Giaco131 opened the pull request.

Giaco131 committed:
Creazione classe Pedina #12

dadebulba commented on back-end/VivaLaDama/VivaLaDama/Models/Pedina.cs:
```diff
@@ -0,0 +1,35 @@
+{
+    public class Pedina
+    {
+        public enum ColorePedina { WHITE, BLACK, NONE }
```
Perchè NONE?

dadebulba reviewed (changes requested):
Di norma in azienda scriviamo tutto in Inglese sia perchè in questo modo c'è consistenza nel linguaggio usato (es. GetOpponentColor suona meglio GetColoreOpponenente) sia perchè tecnicamente chiunque potrebbe mettere mano al tuo codice in futuro.

Giaco131 committed:
Ho tradotto il nome degli attributi e della classe in inglese #20

Giaco131 committed:
Aggiunta metodi #20

Tradotto il nome del metodo GetColoreOpponente in GetOpponentColor e aggiunto il metodo IsColorValid per verificare che il valore di Color appartenga nel dominio definito.

Giaco131 committed:
Ho aggiornato il metodo equals #20

Ho cambiato la struttura del metodo impostando solo un punto di uscita.
